README.md

```diff
@@ -14,7 +14,7 @@ Follow the videos in this and next lessons to implement the page block by block
 - Start writing styles in the `main.scss` file. It is be explained in the `Sass` lesson
 - **DON'T** try to do it `Pixel Perfect` - implement it the most `simple` way so it looks similar;
 - When done check yourself using the [CHECKLIST](./checklist.md) when finished;
-- Deploy and create a Pull Request with a [DEMO LINK](https://<your_account>.github.io/layout_landing-page/)
+- Deploy and create a Pull Request with a [DEMO LINK](https://lmuias.github.io/layout_landing-page/)
```

Comment:
did you deploy the project? The link doesn't work, but I need to see your page to review it. Also please provide it to the PR description. And welcome to ask in the chat all your questions, it'll be much faster
<img width="663" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/98764641/de3b8da1-6749-4604-8eae-28d468a82ac3">